training.rst

```diff
@@ -0,0 +1,113 @@
+Prepare a training on OMERO
+===========================
+
+This chapter describes a setup of a training OMERO.server at 
+your institution. Further, it gives an overview of the other training
+resources as well as hints for trainers about how to present OMERO during a training session.
+
+Resources
+---------
+
+- Community contributions and setups: `Github issue <https://github.com/ome/omero-guides/issues/107>`_.
+
+- `Useful scripts <https://github.com/ome/training-scripts>`_ for setting up the training server.
+
+
+Setup of scripts and OMERO.cli environment
+------------------------------------------
+
+- This guide assumes that you have installed an OMERO.server for training purposes.
+
+- Clone the `training-scripts <https://github.com/ome/training-scripts>`_ repository::
+
+    $ git clone https://github.com/ome/training-scripts.git
```

Comment:
The heading "Setup of scripts" made me expect we'd install them on the server or something, but I see you're just downloading them for use below. So maybe "Download of scripts and OMERO.cli environment setup" ?

Reply:
fixed in https://github.com/ome/omero-guides/pull/236/commits/68522283f4402ea8334b450b705ed8a77c180fc1